tests/profile/test_user_configurable_profiler_v3_batch_request.py

```diff
@@ -1187,3 +1189,48 @@ def test_expect_compound_columns_to_be_unique(
     }
 
     assert expected_expectations == expectations_from_suite
+
+
+@mock.patch("great_expectations.profile.user_configurable_profiler.tqdm")
```

Comment:
Interesting - is this checking to see if the library was called?

Reply:
yup! the mock object has `called` and `called_list` attrs so we can use those to investigate the behavior without actually using the lib